artificial-scrum-master/apps/artificial.scrum.master.frontend/src/app/features/sprint-stats/services/sprint-stats.service.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { GetSprintStats } from '../models/GetSprintStats';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SprintStatsService {
+  private readonly httpClient = inject(HttpClient);
+
+  private readonly baseApiUrl = 'api/projects/sprints';
+
+  public getSprintStats(
+    projectId: number,
+    sprintId: number
+  ): Observable<GetSprintStats> {
+    let params = new HttpParams();
```

Comment:
łot. Ja to bym użył string interpolation ale zapytaj się Filipa co on o tym myśli bo może tak jest zalecane